### ISSUE TITLE
Logging No Longer Works With (at least) Origin Titles

### ISSUE THREAD

**Issue by gardotd426** [flightlessmango/MangoHud#132]
So logging always used to work for me, I have like 10 benchmarks at flightlessmango.com for like 3 different games, and I have like 40 benchmark log files that I haven't uploaded that are on my system right now. However, after updating, I can't get any logging to happen. The HUD works fine, F4 reloads the config fine (I can tell from the Lutris log), F12 toggles the HUD fine. But F2 does nothing, and F2 is the keybind set in my .conf file. I've not changed literally anything at all, it's just completely stopped working. I don't know if it's quit for non-Origin games, but I can tell you it doesn't work for vkcube either, though the HUD works the same it does on Origin (other F-keys work, HUD works in general, etc.). 

Arch Linux, 5.6.2-5-tkg-pds, RX 5600 XT, tried both AMDVLK and RADV. 

MangoHud.conf:
```
cat ~/.config/MangoHud/MangoHud.conf
### MangoHud configuration file
### Uncomment any options you wish to enable. Default options are left uncommented
### Use some_parameter=0 to disable a parameter (only works with on/off parameters)
### Everything below can be used / overridden with the environment variable MANGOHUD_CONFIG instead

################ PERFORMANCE #################

### Limit the application FPS
# fps_limit=

### VSYNC [0-3] 0 = adaptive; 1 = off; 2 = mailbox; 3 = on
 vsync=1

################### VISUAL ###################

### Display the current CPU information
cpu_stats
# cpu_temp

### Display the current GPU information
gpu_stats
# gpu_temp

### Display the frametime line graph
frame_timing

### Display the current system time
# time

### Change the hud font size (default is 24)
font_size=24

### Change the hud position (default is top-left)
position=top-left

### Display the current CPU load & frequency for each core
# core_load

### Display system ram / vram usage
# ram
# vram

### Disable / hide the hud by deafult
# no_display

### Hud position offset
# offset_x=
# offset_y=

### Hud dimensions
# width=
# height=

### Hud transparency / alpha
background_alpha=0.5

### Crosshair overlay (default size is 30)
# crosshair
# crosshair_size=
# crosshair_color=RRGGBB

################## INTERACTION #################

### Change toggle keybinds for the hud & logging
toggle_hud=F12
toggle_logging=F2
```

**Comment by gardotd426** [comment]
Update, doesn't work with Steam titles either (tried Borderlands 3)

**Comment by flightlessmango** [comment]
the `MANGOHUD_OUTPUT` env var was changed to `output_file` param instead, is that the issue? `MANGOHUD_CONFIG=output_file=/path/to/file`

**Comment by gardotd426** [comment]
Ah, the README still says MANGOHUD_OUTPUT, I didn't know there was a
change. I'll see if it works.

**Comment by flightlessmango** [comment]
@gardotd426 did it resolve your issue?

**Comment by gardotd426** [comment]
Oh lord, I think I'm losing my mind, I thought I'd replied already, but apparently I had just typed a draft and never sent it. 

Yes, removing `MANGOHUD_OUTPUT=/path/to/file` with `MANGOHUD_CONFIG=output_file=/path/to/file` seems to work for Borderlands 3, which is on Steam, and I haven't yet gotten to test it with Titanfall 2 on Origin, but given what you've said I imagine this will fix the issue for that game as well. 

I'll go ahead and close this, maybe consider removing the part of the README that says `MANGOHUD_CONFIG=` can be used for logging, since it not longer works, like I said the first thing I did when I saw it wasn't working was check and see if there had been any obvious changes to the README, and I do see now where there's the little `output_file` option in the list of config options, it still has the `MANGOHUD_CONFIG` option under the "Logging" section of the README, so that's rather confusing. But yeah, closing, thanks so much for the help and your contributions to the community.
